pkg/ccl/changefeedccl/protected_timestamps_test.go

```diff
@@ -451,29 +453,74 @@ func TestPTSRecordProtectsTargetsAndSystemTables(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
 
-	s, db, stopServer := startTestFullServer(t, feedTestOptions{})
+	// skip.UnderStress(t, "under stress it can take forever for the span config reconciler to update")
+
+	ctx := context.Background()
+
+	// Useful for debugging.
+	require.NoError(t, log.SetVModule("spanconfigstore=2,store=2,reconciler=2,mvcc_gc_queue=2,kvaccessor=2"))
```

Comment:
Do we still need this?

Reply:
We don't need it, but if it flakes again i'd like to have them.